services/eth_gateway.go

```diff
@@ -576,9 +576,9 @@ func claimUnusedPRLs(completedUploads []models.CompletedUpload) error {
 	// Contract claim(address _payout, address _fee) public returns (bool success)
 	for _, completedUpload := range completedUploads {
 		//	for each completed upload, get its PRL balance from its ETH
-		//	address (completedUpload.ETHAddr) by calling CheckBalance.
+		//	address (completedUpload.ETHAddr) by calling CheckETHBalance.
 		ethAddr := common.HexToAddress(completedUpload.ETHAddr)
-		balance := checkBalance(ethAddr)
+		balance := checkETHBalance(ethAddr)
```

Comment:
I think this one should have been checkPRLBalance.

Reply:
Done